Sample Applications/WPFGallery/MainWindow.xaml

```diff
@@ -104,7 +104,8 @@
                 Background="Transparent"
                 BorderBrush="Transparent"
                 Command="{Binding ViewModel.BackCommand}"
-                WindowChrome.IsHitTestVisibleInChrome="True">
+                WindowChrome.IsHitTestVisibleInChrome="True"
+                ToolTipService.ToolTip="Back">
```

Comment:
```suggestion
                ToolTip="Back">
```

Reply:
@pchaurasia14 ToolTipService is used in other parts of the code without utilizing it other parameters. Reference: ColorTile.xaml - Line No: 59. So, to maintain code uniformity, I have used the ToolTipService in this case instead of ToolTip.

Reply:
It's okay for now.